packages/react-native/ReactCommon/react/nativemodule/core/platform/ios/ReactCommon/RCTTurboModule.mm

```diff
@@ -208,18 +208,32 @@ id convertJSIValueToObjCObject(jsi::Runtime &runtime, const jsi::Value &value, s
 
   jsi::Value error = createJSRuntimeError(runtime, "Exception in HostFunction: " + reason);
   error.asObject(runtime).setProperty(runtime, "cause", std::move(cause));
-  return {runtime, std::move(error)};
+  return jsi::JSError(runtime, std::move(error));
+}
```

Comment:
Is this method now unused? Can it be removed?

Reply:
It's used here https://github.com/krystofwoldrich/react-native/blob/41fac96681d837c80f4db8ee0ee4351d90d40b65/packages/react-native/ReactCommon/react/nativemodule/core/platform/ios/ReactCommon/RCTTurboModule.mm#L500